src/app/shared/changecolor.directive.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { ElementRef } from '@angular/core';
+import { ChangecolorDirective } from './changecolor.directive';
+
+describe('ChangecolorDirective', () => {
+  it('should create an instance', () => {
+    const mockElementRef = new ElementRef(null)
+    const directive = new ChangecolorDirective(mockElementRef);
+    expect(directive).toBeTruthy();
```

Comment:
Wouldn't like to test if it is actually changing the color?

Reply:
Please I will raise another PR for that one had an issues at my end that's why.